### ISSUE TITLE
[Feature Request] File history

### ISSUE THREAD

**Issue by Aurnytoraink** [Rafostar/clapper#115]
Be able to open the last opened file or streamable link

**Comment by frandavid100** [comment]
Agreed. After watching half a movie and then closing Clapper, it would be cool to just be able to open the app again to keep watching, without having to hunt down the file.

**Comment by Rafostar** [comment]
By default file chooser displays recently played media files when opened. Keeping a track of "streamable links" will be much harder through. Excluding the request for URIs that we indeed do not track ATM, can someone please do a brief explanation to me how would requested "File history" be different from what is displayed with "Open File..." menu option?
